Automation_Tools/Selenium-Automation/README.md

```diff
@@ -0,0 +1,67 @@
+## Automation Tool for Web Scraping using Selenium
+
+### 🎯 Goal
+
+We will be making a list of disney movies from IMDB scraped using Selenium. [IMDB]("https://www.imdb.com/list/ls026785255/") is one of the modern websites that uses javascript for dynamic loading, making it hard for beautifulSoup to scrape, But Selenium got us covered. 
+---
+### 🧾 Description
+
+*Selenium* is a powerful python library for automation of web browser
+You can install it using
+`pip install selenium`
+or if you are using anaconda
+`conda install -c conda-forge selenium`.
+
+Then you need to download a web-driver, in chrome it's ChromeDriver, in firefox it's geckodriver.
+Here you will get the links to drivers https://pypi.org/project/selenium/ .
+
+In my case I am using chromedriver (Remember the version of your chrome driver should match up with the version of chrome you are using)
+and once can use any web browser like firfox, safari, edge, of your choice...
+
+As for chromedriver, you can go to `https://chromedriver.chromium.org/downloads`
```

Comment:
Hey @nidhi2026 ,
Specifying this link is enough, you dont have to attach the .exe file into the folder. Please remove it and will review it then.
Thanks!